lib/generate-index.js

```diff
@@ -102,9 +103,30 @@ function generateIndex (playbook, pages, contentCatalog, env) {
       }
     })
 
+
+  const languages = env.DOCSEARCH_LANGS ?
+                    env.DOCSEARCH_LANGS.split(',') :
+                    ['en']
+
+  languages.forEach(language => {
+
+    if (language === 'ja') {
+      // require('../tinyseg')(lunr) // needed for Japanese Support (manually added in https://github.com/MihaiValentin/lunr-languages repository)
```

Comment:
I'm not sure I understand this comment. Do we need to do something special to support Japanese?
I believe that `tinyseg.js` is part of the `lunr-languages` package so we could do the following no?

```js
require('lunr-languages/tinyseg')(lunr)
```